nf_core/modules/bump_versions.py

```diff
@@ -46,8 +46,12 @@ def __init__(
         self.tools_config: Optional[NFCoreYamlConfig]
 
     def bump_versions(
-        self, module: Union[str, None] = None, all_modules: bool = False, show_uptodate: bool = False
-    ) -> None:
+        self,
+        module: Union[str, None] = None,
+        all_modules: bool = False,
+        show_uptodate: bool = False,
+        _dryrun: bool = False,
```

Comment:
adding dryrun was _much_ easier to mock and test, so I went this route.

Reply:
good idea in general. I added it as a parameter